### ISSUE TITLE
Shutdown Time

### ISSUE THREAD

**Issue by sLimLong** [dmustanger/7dtd-ServerTools#415]
Server restart does not work correctly at a certain time. 
After restarting the server, it starts the restart again
`<Tool Name="Shutdown" Enable="True" Countdown="10" Time="06:00,09:00,12:00,15:00,18:00,21:00,01:00,04:00" Alert_On_Login="False" Alert_Count="1" />`

**Comment by ObsComp** [comment]
Okay I will have to try that out. I wonder if it detects the time being passed 06:00 and runs even though it has already run... or not. It needs something to confirm the range is within 10 seconds of the que time I think. I will have to work on that later

**Comment by TheEhTeam** [comment]
Can also Confirm similar results for 
 <Tool Name="Shutdown" Enable="True" Countdown="15" Time="23:45, 11:45" Alert_On_Login="True" Alert_Count="1" />
It enters a continuous loop of shut downs.

**Comment by ObsComp** [comment]
This should now be fixed in version 20.6.8
Please give it a try and let me know if it persists for you

**Comment by TheEhTeam** [comment]
After updating it seems changing/enabling shutdown has not immediately put it into a shutdown loop as the previous version did. I will let you know how it goes after a full 24hr cycle with two designated shutdowns. Thank you

**Comment by TheEhTeam** [comment]
Cheers the first shutdown after enabling was missed but the next day it seems to work as intended with multiple shutdowns. Thanks again. Confirmed fixed.

**Comment by sLimLong** [comment]
Restarts somehow work strangely. 48 hours have passed since the server was launched. There were only 2 restarts
```
    <Tool Name="Shutdown" Enable="true" Countdown="10" Time="06:00,09:00,12:00,15:00,18:00,21:00,01:00,04:00" Alert_On_Login="False" Alert_Count="1" />
    <Tool Name="Shutdown_Extended" UI_Lock="False" Interrupt_Bloodmoon="False" />
```

**Comment by ObsComp** [comment]
Alright I will try a long term test with that one. It will take a few hours to run. Maybe something still does not function in the event scheduler logic
________________________________

**Comment by ObsComp** [comment]
This should be fixed in version 20.6.9. Thank you for your report. If any problems persist, please do not hesitate to let me know
